Epsilon.Abstractions/Components/IPageComponentManager.cs

```diff
@@ -1,8 +1,6 @@
 namespace Epsilon.Abstractions.Components;
 
-public interface IPageComponentManager
+public interface IPageComponentManager: IComponentManager
```

Comment:
At this point, the component becomes a manager, because you're adding the create or update methods.